lib/src/toggle-group/ToggleGroup.jsx

```diff
@@ -51,45 +54,82 @@ const DxcToggleGroup = ({
+        <OptionsContainer id={toggleGroupId} role={multiple ? "group" : "radiogroup"}>
+          {options.map((option, i) => (
+            <ToggleContainer
+              selected={
+                multiple
+                  ? value
+                    ? value.includes(option.value)
+                    : selectedValue.includes(option.value)
+                  : value
+                  ? option.value === value
+                  : option.value === selectedValue
+              }
+              role={multiple ? "switch" : "radio"}
+              aria-checked={
+                multiple
+                  ? value
+                    ? value.includes(option.value)
+                    : selectedValue.includes(option.value)
+                  : value
+                  ? option.value === value
+                  : option.value === selectedValue
+              }
+              tabIndex={!disabled ? tabIndex : -1}
+              onClick={() => !disabled && handleToggleChange(option.value)}
+              isFirst={i === 0}
+              isLast={i === options.length - 1}
+              isIcon={option.iconSrc || option.icon}
+              optionLabel={option.label}
+              disabled={disabled}
+              onKeyPress={(event) => {
+                handleKeyPress(event, option.value);
+              }}
+              key={`toggle-${i}-${option.label}`}
+            >
+              <OptionContent>
+                {option.icon && (
+                  <IconContainer optionLabel={option.label}>
+                    {typeof option.icon === "object" ? option.icon : React.createElement(option.icon)}
+                  </IconContainer>
+                )}
+                {option.iconSrc && <Icon src={option.iconSrc} optionLabel={option.label} />}
+                {option.label && <LabelContainer>{option.label}</LabelContainer>}
+              </OptionContent>
+            </ToggleContainer>
+          ))}
+        </OptionsContainer>
       </ToggleGroup>
     </ThemeProvider>
   );
 };
 
+const Label = styled.label`
+  color: ${(props) => (props.disabled ? props.theme.disabledLabelFontColor : props.theme.labelFontColor)};
+  font-family: ${(props) => props.theme.labelFontFamily};
+  font-size: ${(props) => props.theme.labelFontSize};
+  font-style: ${(props) => props.theme.labelFontStyle};
+  font-weight: ${(props) => props.theme.labelFontWeight};
+  line-height: ${(props) => props.theme.labelLineHeight};
+`;
+
+const HelperText = styled.span`
+  color: ${(props) => (props.disabled ? props.theme.disabledHelperTextFontcolor : props.theme.helperTextFontColor)};
+  font-family: ${(props) => props.theme.helperTextFontFamily};
+  font-size: ${(props) => props.theme.helperTextFontSize};
+  font-style: ${(props) => props.theme.helperTextFontStyle};
+  font-weight: ${(props) => props.theme.helperTextFontWeight};
+  line-height: ${(props) => props.theme.helperTextLineHeight};
+`;
+
 const ToggleGroup = styled.div`
+  display: flex;
```

Comment:
There are two values for the attribute `display`: `inline-text` and `flex`.

Reply:
Solved!